bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/package.scala

```diff
@@ -96,15 +92,17 @@ case class ObservationInfo(ctx: Option[ObsContext], objectName: Option[String],
 }
 
 object ObservationInfo {
+  val DefaultInstrument = SPComponentType.INSTRUMENT_VISITOR
+
   // Observation context loaded initially with default parameters
-  // TODO don't hardcode to Gmos South
-  val zero = new ObservationInfo(None, "".some, Coordinates.zero.some, SPComponentType.INSTRUMENT_GMOSSOUTH.some, None, Nil, SPSiteQuality.Conditions.BEST.some, UCAC4, ProbeLimitsTable.loadOrThrow())
+  val zero = new ObservationInfo(None, "".some, Coordinates.zero.some, DefaultInstrument.some, None, Nil, SPSiteQuality.Conditions.BEST.some, UCAC4, ProbeLimitsTable.loadOrThrow())
 
-  private val InstMap = Map[String, SPComponentType](
+  val InstList = List(
```

Comment:
We really need something in the core that allows to get a list of available instruments...